tests/constraints.txt

```diff
@@ -1,12 +1,13 @@
 cattrs==22.2.0
+hatchling==0.25.0
```

Comment:
Is this to support Python 2? If so, note that there is a patch release after this one.

Reply:
No, 3.7 is our oldest. Was just trying to pick the oldest one easy to support as a minimum. Can change if you'd like.